project/src/main/career/career-data.gd

```diff
@@ -50,6 +58,21 @@ const RANK_MILESTONE_FAIL := {"rank": 64.0, "distance": 0, "color": Color("babab
 const DAILY_STEPS_GOOD := 25
 const DAILY_STEPS_OK := 8
 
+## time to show when the number of levels the player has played is an unexpected value
+var invalid_time_of_day := tr("?:?? zm")
+
+## key: (int) corresonding to the number of levels the player has played
+## value: (String) human-readable time of day
+var time_of_day_by_hours := {
+	0: tr("11:00 am"),
+	1: tr("12:10 pm"),
+	2: tr("1:20 pm"),
+	3: tr("6:30 pm"),
+	4: tr("7:40 pm"),
+	5: tr("8:50 pm"),
+	6: tr("10:00 pm"),
+}
```

Comment:
Leave a comment about this moving

Reply:
Expanded commit comment, `Moved time_of_day_by_hours from CareerTimeLabel into CareerData`